src/averageAges.js

```diff
@@ -55,7 +69,22 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const arrayOfKids = (!onlyWithSon)
```

Comment:
You can simplify the ternary operator by removing the parentheses around `!onlyWithSon`. Use `const arrayOfKids = !onlyWithSon ? ... : ...` instead.